test/loader/test_link_neighbor_loader.py

```diff
@@ -166,7 +164,7 @@ def test_link_neighbor_loader_edge_label():
     )
 
     for batch in loader:
-        assert batch.edge_label.dtype == torch.float
+        assert batch.edge_label.dtype == torch.long
```

Comment:
Can we make this a `float`? It simplifies usage of `BCELoss`.

Reply:
Ah i see, then maybe we want to convert `torch.float` in the `init` function.